### ISSUE TITLE
Scrollable areas resize randomly

### ISSUE THREAD

**Issue by joshkim81** [inndevs/mac-taskbar-public#10]
The space inside scrollable areas (text boxes, window panes, etc) will randomly resize and then revert back repeatedly throughout the day. It almost looks "glitchy" and it's hard to capture since it seems so random and even happens to windows not in focus. I think I've seen it happen on every app that I use. I can't say for sure that this is related to the app but it seems to have started after I installed it. I typically run at least one other monitor on my 2023 Mac Air.

**Comment by schneidermayer** [comment]
Hi, puh, hard to tell what you mean, I've never heard of this problem. Where are those scrollable areas? In your browser or other apps? What OSX version are you using? I video would be very helpful. But I doubt it has to do with the taskbar.

**Comment by joshkim81** [comment]
Yes, in the browser and other apps - it's any area that has a scrollbar. I'm on Sonoma 14.2.1. I'll try to get a video of it. 

I thought it might be this app since it resizes windows after maximizing to fit the taskbar underneath. It felt like maybe that was related but you're probably right!

**Comment by schneidermayer** [comment]
Ah, you could also be right about the resizing thing. Mac Taskbar always resizes windows if the go beneath the taskbar. If you could the video, or a way to reproduce this, it would help a lot.

**Comment by joshkim81** [comment]
I'll try to get one for you
